frontend/plugins/insurance_ui/src/modules/insurance/components/vendorUsers/VendorUsersMoreColumn.tsx

```
@@ -11,11 +12,12 @@ export const VendorUsersMoreColumn = ({
   onEdit: (user: any) => void;
   onRefetch: () => void;
 }) => {
+  const { t } = useTranslation('insurance');
   const user = cell.row.original;
   const { deleteVendorUser } = useDeleteVendorUser();
 
   const handleDelete = async () => {
-    if (confirm('Are you sure you want to delete this user?')) {
+    if (confirm(t('confirm-delete-user'))) {
```

ESLint

[error] Unexpected use of 'confirm'.

(no-restricted-globals)